apps/studio/src/pages/sites/[siteId]/users.tsx

```diff
@@ -0,0 +1,114 @@
+import { useContext } from "react"
+import { Box, HStack, Text, VStack } from "@chakra-ui/react"
+import { Button } from "@opengovsg/design-system-react"
+import { ResourceType } from "~prisma/generated/generatedEnums"
+import { BiPlus } from "react-icons/bi"
+import { PiUsersBold } from "react-icons/pi"
```

Comment:
[comment] oh what, i thought we were using `bi` HAHAH

Reply:
yeah i couldn't find an equilvalent from `bi` and this is the closest i can find in terms of the looks 😢